index.html

```diff
@@ -111,5 +111,31 @@
   </ul>
 </div>   
     <!-- Most Recent html ends here: --> 
+    <!-- FOOTER BEGINS HERE: -->
+<div class="footer">
+  <img class="facebook" src="assets/facebook.png">
+  <img class="instagram" src="assets/instagram.png">
+  <img class="pinterest" src="assets/pinterest.png">
+  <img class="twitter" src="assets/twitter.png">
+  <div class="table">
+<ul>
+  <li><a href="#about us">About Us</a></li>
+  <li><a href="#news">Contact Us</a></li>
+  <li><a href="#contact">Subscriptions</a></li>
+  <li><a href="#about">Media Kit</a></li>
+  <li><a href="#about">Privacy</a></li>
+  <li><a href="#about">Where to Buy</a></li>  
+</ul>
+<ul>
+  <li><a href="#digital edition">Digital Edition</a></li>
```

Comment:
What are these for? Are these links that were on the original footer? If possible, we should make sure that the links we have on our page are ones that we can either link to another spot on our page and only have ones that don't go outside of our site. Thanks!